graylog2-server/src/main/java/org/graylog2/contentpacks/model/entities/WidgetEntity.java

```diff
@@ -142,4 +166,103 @@ public WidgetDTO toNativeEntity(Map<String, ValueReference> parameters, Map<Enti
         }
         return widgetBuilder.build();
     }
+
```

Comment:
The same applies as for `DashboardWidgetConverter`: `I have read through the remainder of this class, but feel unable to really review it in depth in a reasonable time frame, as it's a huge bunch of special case logic for the various widget types. Gotta trust you here. :)`